saleor/graphql/shop/mutations.py

```diff
@@ -298,6 +298,11 @@ class OrderSettingsUpdateInput(graphene.InputObjectType):
         "will be marked as unconfirmed. When enabled orders from checkout will "
         "become unfulfilled immediately.",
     )
+    automatically_fulfill_non_shippable_gift_card = graphene.Boolean(
+        required=True,
+        description="When disabled, all new non shippable gift cards "
+        "will require manual fulfillment.",
```

Comment:
I am not sure if the description is good, as when is disabled all gift cards order will require manual fulfillment. Maybe we should describe it from the other side, like:
`When enabled, all non-shippable gift card orders will be fulfilled automatically.`.